median/examples/simp/src/lib.rs

```diff
@@ -33,19 +33,21 @@ median::external! {
         //create an instance of your object
         //setup inlets/outlets and clocks
         fn new(builder: &mut dyn MaxWrappedBuilder<Self>) -> Self {
+            builder.with_default_inlet_assist(&"first inlet");
```

Comment:
Running `cargo clippy` suggests that this is a needless borrow and you can just use `builder.with_default_inlet_assist("first inlet")`

Reply:
Also on line 41, 44, and 50